### DISCUSSION TITLE
Adwaita themes for labwc

### DISCUSSION THREAD

**Discussion by davidphilipbarr** [labwc/labwc#1775] [Show and tell]
I've made some reasonable accurate/passable (in my opinion) adwaita based themes for people who use gnome/ libadwaita apps and therefore i suppose people not that interested in themes?  I've made pictures and everything. Most of the apps i use have csd headers so it's rare-ish i see these, but they may be of use to someone. 

https://github.com/davidphilipbarr/labwc-dump/tree/main/themes

**Comment by johanmalm** [comment]
Wow! Really nice. We ought to link to that from the website.

**Comment by 01micko** [comment]
> Wow! Really nice. We ought to link to that from the website.

.. and also `labwc-artwork`.

**Reply by davidphilipbarr** [reply]
Yeah I have a branch for labwc-artwork, I'm not sure how to structure them, at the moment I have the SVG buttons split from *-labwc themes, which I think makes sense- but I'm still tinkering.
